core/designsystem/src/main/java/com/susu/core/designsystem/component/textfield/PriceVisualTransformation.kt

```diff
@@ -0,0 +1,37 @@
+package com.susu.core.designsystem.component.textfield
+
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+import androidx.compose.ui.text.input.VisualTransformation
+import java.text.DecimalFormat
+
+class PriceVisualTransformation : VisualTransformation {
+    override fun filter(text: AnnotatedString): TransformedText {
+        val amount = text.text
+        val formatAmount = if (amount.isEmpty()) "" else DecimalFormat("#,###").format(amount.toLong())
```

Comment:
```suggestion
        val formatAmount = if (amount.isEmpty()) "" else DecimalFormat("#,###").format(amount.to BigDecimal())
```

이렇게하면 Long보다 큰 숫자 입력받을 수 있어용

Reply:
적용했습니다~ 이제 우다다 큰 수를 입력하면 터지지는 않게 되었습니다